packages/smooth_app/lib/widgets/ranking_floating_action_button.dart

```diff
@@ -24,13 +24,27 @@ class RankingFloatingActionButton extends StatelessWidget {
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
             SizedBox(width: MediaQuery.of(context).size.width * 0.09),
-            FloatingActionButton.extended(
-              heroTag: 'ranking_fab_${Random(100)}',
-              elevation: 12.0,
-              icon: const Icon(rankingIconData),
-              label: Text(AppLocalizations.of(context).myPersonalizedRanking),
-              onPressed: onPressed,
-            ),
+            Flexible(
```

Comment:
Generally speaking I'm not sure we need that `Row` and especially that `SizedBox`.